source-oracle/backfill.go

```diff
@@ -132,13 +142,28 @@ func (db *oracleDatabase) ScanTableChunk(ctx context.Context, info *sqlcapture.D
 		}
 		resultRows++
 		rowOffset++
+
+		// Keyless backfills are not ordered, so we need to check every row to see if it includes the last ROWID known
+		// at the time of starting the backfill which is the last rowid in the backfill range slice
+		if state.Mode == sqlcapture.TableStateKeylessBackfill && !backfillComplete {
+			backfillComplete = rowid >= lastTableRowID
```

Comment:
What happens here if the user starts a backfill and then drops the last row in the table?

Reply:
I already changed this logic while you were reviewing this, I think the new logic is fine since we now just check whether we were capturing the last range, and if so, we mark `backfillComplete = true` by comparing `rowidRangeEnd` with `lastTableRowID`